File: desktop/src/gdx/scala/demo/desktop/DesktopLauncher.java
package gdx.scala.demo.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import gdx.scala.demo.GdxScalaDemoGame;
import gdx.scala.demo.SuperKoalio;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
//		new LwjglApplication(new SuperKoalio(), config);
		new LwjglApplication(new GdxScalaDemoGame(), config);
//		new LwjglApplication(new MeshTutorial1(), config);
	}
}

File: desktop/src/gdx/scala/colordash/desktop/DesktopLauncher.scala
package gdx.scala.colordash.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import gdx.scala.colordash.ColorDashGame

object DesktopLauncher {
  def main(arg: Array[String]) {
    val config: LwjglApplicationConfiguration = new LwjglApplicationConfiguration
    new LwjglApplication(ColorDashGame, config)
  }
}